ddtrace/llmobs/_experiment.py

```diff
@@ -92,7 +101,7 @@ def __init__(
         self._evaluators = evaluators
         self._description = description
         self._tags = tags or []
-        self._config: Dict[str, Any] = config or {}
+        self._config: Dict[str, JSONType] = config or {}
```

Comment:
do the contents actually need to be serializable? do we submit them to the backend? I actually prefer we keep it stricter for the time being but was curious